calendar/types.bal

```diff
@@ -0,0 +1,666 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+# Represents configuration parameters to create Microsoft Calendar.
+#
+# + clientConfig - OAuth client configuration
```

Comment:
OAuth 2.0

Reply:
Hi Shani, 

This has been updated with new changes in record type [1].
Please refer to the new record documentation here [2].

[1] https://github.com/ballerina-platform/module-ballerinax-microsoft.outlook.calendar/blob/4e2c2ad018816ffe0a3fee9c205f988472f2afe0/calendar/client.bal#L31

[2] https://github.com/ballerina-platform/module-ballerinax-microsoft.outlook.calendar/blob/4e2c2ad018816ffe0a3fee9c205f988472f2afe0/calendar/types.bal#L19

Thanks